docs/package.json

```diff
@@ -3,18 +3,20 @@
   "version": "1.0.0",
   "description": "",
   "main": "next.config.js",
+  "type": "module",
   "scripts": {
-    "start": "next dev",
-    "build": "next build && next export"
+    "dev": "next dev",
```

Comment:
The `README.md` for `docs` has instructions to run the `dev` command, so I moved `next dev` to use that command instead of `start`.